### DISCUSSION TITLE
Cyls > 80 problem

### DISCUSSION THREAD

**Discussion by kivijakola** [keirf/flashfloppy#759] [Q&A]
Hi,
I'm trying to write to a raw .img image that has following format:

cyls = 83
heads = 2
secs = 10
bps = 1024

I have empty .img file stored on usb stick and writing is working fine up to the cylinder 80. After that I always get Windows message "The drive cannot find the sector requested" Writing is working fine with standard floppy drive and diskette.

There is not any error message in uart output either. I couldn't spot the place in source code that is triggering this error. 

I'm using SAMdisk for writing: SAMdisk.exe a: hunter.dsk -s10 -z3

Any hint how to get over this?

**Comment by keirf** [comment]
Is the empty .img file the correct size? `83*2*10*1kB = 1660kB = 1699840 bytes`
